openslides_backend/action/actions/motion/create.py

```diff
@@ -90,6 +90,14 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 raise ActionException(
                     "You can't give amendment_paragraph_$ in this context"
                 )
+        # if lead_motion use category_id and block_id from the lead_motion
+        if instance.get("lead_motion_id"):
```

Comment:
In general, this should only be done if the request user does not have `motion.can_manage`. Please add a test creating an amendment as an admin (having `motion.can_manage`) and setting `category_id` and `block_id` to something different than the lead motion. this must succeed.

Change the test slightly, that the request user does not have `motion.can_manage`: It must still succeed, but ignore the provided `block_id` and `category_id` and use the data from the lead motion.

Reply:
Changed, if the user doesn't have `motion.can_manage` both values are changed. And updated a test for it.

Reply:
> It must still succeed, but ignore the provided `block_id` and `category_id` and use the data from the lead motion.

Shouldn't the request be denied because of missing permissions in this case? You try to set fields which you are not allowed to set. We said we didn't want to silently ignore parts of the payload.

Reply:
Yes, this is right.

Reply:
I don't understand this discussion, what should be done here?

Reply:
Add a test where the request user does not have `motion.can_manage` and tries to create a motion providing `lead_motion_id`, `category_id` and `block_id`. The request should fail because the user does not have permission to update the two latter fields. This has to be adjusted in the code accordingly, currently it is just silently ignored (I think).

Reply:
No, that is not right. The requirement says: If the user doesn't have `motion.can_manage`, then a whitelist should be checked and `category_id` and `block_id` should be in the wl, because of the `lead_motion_id`. Else it doesn't make sense.

Reply:
Yes, as @FinnStutzenstein and I discussed above, this requirement was wrong. We want to throw errors in OS4 if the user sends any invalid input instead of silently ignoring it. I will change the wiki entry.